### DISCUSSION TITLE
终端界面优化：btop提示宽高不够

### DISCUSSION THREAD

**Discussion by gfgjs** [lollipopkit/flutter_server_box#332] [Ideas]
如图，终端btop，因宽高不够而无法使用。能否优化上下标题栏（红框处），使得屏幕可用空间增大？
![IMG_20240411_173022](https://github.com/lollipopkit/flutter_server_box/assets/29412548/3abdeedc-da28-46bf-89ea-925c40db0e7c)

**Comment by lollipopkit** [comment]
使用首页的服务器卡片内的SSH的话，没有下面的tab。

已新增设置：虚拟按键显示为一行。

后继会调整ssh tab的逻辑。
